loops-nate-arnold.js

```diff
@@ -0,0 +1,39 @@
+// Logging values with for loops
+   
+// -Write a for loop that logs each number from 1 - 20.
+    for(let i=1; i<21; i++){
+      console.log(i)
+    }
+// -Write a for loop that logs the result of each number from 1 - 20 tripled.
```

Comment:
Mission accomplished. So this challenge question does not request an array, you can approach it just as you approach the question on Line 7. So you can refactor the lines 4-6 to give you the expected output.
for(let i = 1; i < 21; i++){
  console.log(i * 3)
}